app/models/concerns/hyrax/ability.rb

```diff
@@ -309,9 +309,8 @@ def curation_concerns_permissions
       # user can version if they can edit
       alias_action :versions, to: :update
       alias_action :file_manager, to: :update
-
       return if admin?
-      cannot :index, Hydra::AccessControls::Embargo
+      cannot :manage, Hydra::AccessControls::Embargo
```

Comment:
Need deeper conversation on who exactly should be able to edit an embargo.

Reply:
I've tried to launch the discussion in the originating issue at #4782. 

i think #4801 might be a fix for the test suite issue this PR ran into.